src/docs/development/tools/devtools/cpu-profiler.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Using the CPU profiler view
+description: Learn how to use the DevTools CPU profiler view.
+---
+
+{{site.alert.note}}
+  The CPU profiler view works with Dart CLI and mobile apps only.
+  Use Chrome DevTools to [analyze performance][]
+  of a web app.
+{{site.alert.end}}
+
+## What is it?
+
+The CPU profiler view allows you to record and profile a
+session from your Dart or Flutter application.
+
+{{site.alert.note}}
+  **If you are running a Flutter application,
+  use a profile build to analyze performance.**
+  CPU profiles are not indicative of release performance
+  unless your Flutter application is run in profile mode.
+{{site.alert.end}}
+
+<!-- TODO(kenz): this is the same information that is in the Timeline page.
+Consolidate somehow. -->
```

Comment:
You can always create a _definitions.md or _profiler-def.md page (or something appropriately named, but the filename should begin with an underbar).. Then just include the file in both places.

Reply:
ptal - not sure if the way I included the file is correct. followed the guide here: https://developer.lightbend.com/docs/paradox/current/directives/includes.html#include-block

Reply:
PTAL at the include format - `{% include _profiler.md %}` - is that correct? I cannot build the website on my machine without some configuration tasks so I don't always for "safe" .md changes.

Reply:
Sorry I didn't see this before, @kenzieschmoll. Yes, that include is correct. Can we go ahead and merge this?